admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -28,31 +32,52 @@ export const ApplicationPage: React.FunctionComponent = () => {
     const columns = [ 'Event Type', 'Name', 'Description', 'Event Type Id' ];
 
     const newEvent = useCreateEventType();
-    const [ id ] = React.useState<string | undefined>();
-    const [ displayName, setDisplayName ] = React.useState<string | undefined>();
-    const [ name, setName ] = React.useState<string | undefined>();
-    const [ description, setDescription ] = React.useState<string | undefined>();
+    const [ eventType, setEventType ] = React.useState<Partial<EventType>>({});
+
+    const [ showModal, setShowModal ] = React.useState(false);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [ isEdit, setIsEdit ] = React.useState(false);
+    const [ visableButton, setVisableButton ] = React.useState(false);
```

Comment:
It looks like `isEdit` is not used anywhere and on glance, it could replace or be-replaced-by the `visableButon`. 
Could you double check and remove the unused ones?